earth_enterprise/rpmlintrc

```diff
@@ -0,0 +1,41 @@
+# -- revisit for later review
+
+# Disable to review/remediate later...
+addFilter("E: non-standard-executable-perm")
+
+# Disable to ignore for now, often issues from third party code, some is valid though.
+addFilter("^.* /opt/")
+addFilter("^.* /var/opt/")
+
+# Because this is in init.d it gets LOTS of error hits...
+addFilter("E: .* /etc/init.d/gevars.sh")
```

Comment:
Shouldn't we fix this by moving the file, instead of ignoring it?

Reply:
I had wanted to speak to you about this first.  But yes, we should...

Reply:
I wanted to speak with you about this first before moving it.

Reply:
As far as I'm concerned, I think we should get this file out of `/etc/init.d`.

Actually, it seems that this file currently contains variables used in Fusion, as well as Server.  We could split it, and package each part with the service it's part of. If we don't want to fix everything it once, perhaps, under `/etc/opt/google/opengee-common` would be good place.

Reply:
As you explain below, this would be outside the scope of this pull request, so fixing this would be in a separate ticket.

Reply:
We can introduce a separate issue for that.  I did briefly outline the goals and purpose of this branch in a github ticket, #866